src/pages/RFOX/components/Unstake/hooks/useRfoxUnstake.tsx

```diff
@@ -44,7 +45,7 @@ type UseRfoxUnstakeReturn = {
   unstakeMutation: UseMutationResult<string | undefined, Error, void, unknown>
   isUnstakeTxPending: boolean
   isGetUnstakeFeesEnabled: boolean
-  userStakingBalanceOfQuery: UseQueryResult<string, Error>
+  userStakingBalanceOfQuery: UseQueryResult<string | undefined, Error>
```

Comment:
This was intended yeah because it's totally possible a user who has not yet staked will get an error response which results in `useQueries` retrying all of them due to error, so we actually catch the error and return undefined in this case as the user not staking in that epoch is not an "error" but an "empty" response

Reply:
With the word @0xApotheosis did with custom token (see https://github.com/shapeshift/web/pull/7285/files#diff-f750893454c0095e6cfd30f42d2fcd27fd1d6c2a4bfdf3ed385e5017b481d7eaL37 ) it might be possible to revert this

Reply:
actually, disregard me this is a different change - reverting